meeting-minutes/2023-05-23.md

```diff
@@ -0,0 +1,177 @@
+
+jar: we could handle tc39 meeting as one-off reschedule
+
+markm: yes, one-offs are common in recurring.  and usually tc39 does not happen so early
+
+ian: I am fine tracking DST, or not
+
+tsyesika: would rather not track DST
+
+jar: we have some inertia towards DST
+
+ian: I will propose not tracking DST, does anyone have a *problem* with that
+
+markm: I will abstain
+
+ian: we have one minor preference for not tracking dst
+
+cwebber: as programers we should oppose timezones and daylight savings so we should oppose DST
+
+**Decision:** 19:00 UTC on 2nd Tuesdays of the month.
+
+## TOPIC: Issue closing procedure
+
+jar: dckc proposed structure for closing issues
+
+dckc: we can do a test with closing one of the particular issues as a test
+anyone may (and should!) close an issue by making a comment that
+
+ - clearly states the resolution
+ - cites the position of Spritely, Agoric, and capnproto, with at least 1 in favor and none objecting
+ - closes the issue, subject to appeal, esp. at the next meeting
+
+jar: issue #50 and issue #46.  hearing no objections to this, they remain closed.  And that's it!  And we have these two good examples of how we go about this.  I'll figure out someplace to put it.
+
+## TOPIC: Test suite status repo
+
+### CapTP draft spec
+
+tsyesika: update on where work is: draft CapTP spec, got a lot of updates, I think I've addressed all the feedback to date, though I think Agoric has some review still pending
+
+dckc: we've concluded our review
+
+tsyesika: does anyone have objections to merging it?
+
+dckc: I expect editorial issues to get addressed to our satisfaction in review of this PR.  some technical details that I don't expect we'll resolve now; we made issues for those
+
+markm: ok!
+
+### Test suite
+
+https://github.com/ocapn/ocapn-test-suite f7dc5e3
+
+tsyesika: test suite is up on github, it is written in python, currently supports tor onion netlayers because that's what spritely does but it can be flexible for netlayers.  has a bunch of tests.  one thing is missing is third party handoffs, but planning on adding those in the next week.  and missing op:pick.  has status report, but it can be improved to show positive/negative passing status, I'm sure it can be improved if people have feedback.  After that will do netlayers then implementation report.  by next week hoping to have test suite updated
+
+cwebber: congrats on getting captp test suite done, and good choice getting it done in python
+
+zarutian silently applauds
+
+dckc: is that a real captp implementation in python?
+
+tsyesika: yes, just enough to run the test suite, but it would be a good starting point for implementing ocapn.
+
+dckc: that's impressive, hadn't noticed.  Has anyone else looked?
+
+dthompson: I did look at it when porting code from racket to guile and it was useful for me and neat to look at
+
+dckc: it would be nifty to get a few volunteers are not from spritely to take a look
+
+ian: I can try to give a peek
```

Comment:
Have added @zenhack as a assignee for the test suite issue in ocapn repo. He can remove himself when done. Hoping this will be an effective way to track... I wish we had action items

Reply:
Assigning @zenhack to #8 looks like a great way to track this.